docs/user-guide/command-line-interface.md

```diff
@@ -45,37 +47,44 @@ Specifying rules and plugins:
 
 Fixing problems:
   --fix                          Automatically fix problems
-  --fix-dry-run                  Automatically fix problems without saving the changes to the file system
+  --fix-dry-run                  Automatically fix problems without saving the
+                                 changes to the file system
```

Comment:
Were these whitespace changes intentional?

Reply:
No, I think they happened due to me running `--help` on a narrow terminal window. I can certainly fix these for our docs, if desired, since web browsers could horizontally scroll if needed. Thoughts?

Reply:
I figured it out. Last time I ran `--help` and piped directly into a file, so no TTY and no max-width TTY display. This time, I was using the Node runtime in a TTY. I'll push another commit to fix this.